libs/components/popovers/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.strict.json",
+  "extends": "../../../tsconfig.base.json",
```

Comment:
We'll need to disable strict mode for popovers until `@skyux/indicators` supports strict mode.